src/modules/layoutValidation.js

```diff
@@ -24,25 +24,27 @@ const isAxisValid = (axis) =>
         })
     )
 
-const validateDimension = (dimension, error, requireItems) => {
-    if (!(dimension && dimensionIsValid(dimension, { requireItems }))) {
-        throw error
+export const validateLineListLayout = (layout, { doNotThrow } = {}) => {
```

Comment:
instead of introducing a new prop (`doNotThrow`), couldn't you just catch the error that is being thrown and set a prop to to false instead?
for example something like this in `useDownloadMenu.js`
```
let something = true
try { 
    validateLineListLayout(current) 
} catch (error) {
    if (isVisualizationError(error)) {
        something = false
    } else {
        throw error
    }
}
```

Reply:
The idea is that the validation function now can be used for dry runs. We want to instruct the function when we call it.

Reply:
oh ok, adding the dry-run option seems a bit over-engineered, when you can just catch the error in the consumer if you don't want it to throw. But if you insist then I would at least call it `dryRun` (or something similar) instead of `doNotThrow`...

Reply:
@martinkrulltott I missed the point of your first comment. I'll revisit it.

Reply:
Ok, I'll rename to `dryRun`. Technically the function doesn't return errors, so the dry run returns true or false while a normal run returns true or throws an error. I think it's nice to be able to do a dry run without trycatching.